.gitignore

```diff
@@ -15,6 +15,8 @@ networktables.json
 ntcore/connectionlistenertest.json
 ntcore/timesynctest.json
 
+nanopb_pb2.py
+
```

Comment:
I'm assuming this is a temporary change? If so, this is a reminder to remove it prior to merging.